tests/services/asset_import/test_import_common.py

```diff
@@ -55,6 +57,21 @@ def test_cannot_upload_raw_text_to_pdf_project(self, *_):
         with self.assertRaises(FileNotFoundError):
             import_assets(self.kili, self.project_id, assets, disable_tqdm=True)
 
+    @patch.object(ProjectQuery, "__call__", side_effect=mocked_project_input_type("TEXT"))
+    @patch.object(AssetQuery, "count", return_value=1)
+    def test_return_the_ids_of_created_assets(self, *_):
+        assets = [{"content": "One"}, {"content": "Two"}, {"content": "Three"}]
+
+        def graphql_execute_side_effect(*args, **kwargs):
```

Comment:
this is a hack until we get the new architecture in place correct?

Reply:
yes, it will be much easier after!